File: cpp/3rd_Week/MariMencoba3/PenjualBahanBangunan.cpp
#include "Manusia.cpp"
#include "TokoBangunan.cpp"

using namespace std;

class PenjualBahanBangunan: public Manusia, public TokoBangunan {
    private:
        string namaPenjual;

//    constructor
    public:
        PenjualBahanBangunan(){}

    //    set and get method
        string getNamaPenjual() {
            return namaPenjual;
        }

        void setNamaPenjual(string namaPenjual) {
            this->namaPenjual = namaPenjual;
        }

        ~PenjualBahanBangunan(){};
};

File: java/javadpbo/src/com/sem4dpbo/week3/marimencoba3/PenjualBahanBangunan.java
package com.sem4dpbo.week3.marimencoba3;

public class PenjualBahanBangunan extends TokoBangunan {
    private String noKtp, namaPenjual;

//    constructor
    public PenjualBahanBangunan(){}

//    set and get method
    public String getNoKtp() {
        return noKtp;
    }

    public void setNoKtp(String noKtp) {
        this.noKtp = noKtp;
    }

    public String getNamaPenjual() {
        return namaPenjual;
    }

    public void setNamaPenjual(String namaPenjual) {
        this.namaPenjual = namaPenjual;
    }
}
